docs/guides/basic-rbac.mdx

```diff
@@ -105,55 +110,57 @@ declare global {
   const { sessionClaims } = auth()
-
   return sessionClaims?.metadata.role === role
 }
 ```
 
-This `checkRole()` helper will accept a role using the `Roles` type and will return `true` if the user has that role, or `false` if the user does not.
+This `checkRole()` helper will accept a role of type `Roles` and return `true` if the user has that role, or `false` if the they do not.
+
+Next, refactor the admin dashboard to use the `checkRole()` helper.
 
-Next, the admin dashboard can be refactored to use the `checkRole()` helper. Navigate back to the admin dashboard file. In the `if()` statement, remove the check that was used previously and replace it with the new `checkRole()` helper with `"admin"` as the argument.
+1. Navigate back to the admin dashboard file.
+1. In the `if()` statement, remove the previous role check.
+1. Replace it with the new `checkRole()` helper, passing `"admin"` as the argument.
 
 ```tsx {{ filename: 'app/admin/dashboard/page.tsx', mark: [2, [5, 8]] }}
 import { redirect } from 'next/navigation'
 import { checkRole } from '@/utils/roles'
 
-export default function AdminDashboard() {
+export default async function AdminDashboard() {
   // If the user does not have the admin role, redirect them to the home page
   if (!checkRole('admin')) {
     redirect('/')
   }
 
-  return (
-    <>
-      <h1>This is the admin dashboard</h1>
-      <p>This page is restricted to users with the 'admin' role.</p>
-    </>
-  )
+  return <p>This is the protected admin dashboard restricted to users with the `admin` role.</p>
 }
 ```
 
 > [!NOTE]
-> You can modify the behavior of the helper function to meet your needs. Maybe it will return the roles that the user has, or you could create a `protectByRole()` and have that function handle the redirect.
+> You can customize the behavior of the `checkRole()` helper function to suit your needs. For example, you could modify it to return the roles a user has or create a `protectByRole()` function that handles role-based redirects.
 
 ## Add admin tools to find users and add roles
 
-You can leverage the `checkRole()` function you added along with server actions to build basic tools for finding users and managing roles.
+You can use the `checkRole()` function along with server actions to build basic tools for finding users and managing roles.
 
-Start with the server action. The `setRole()` action below will check the role of the current user using the `checkRoles()` helper to confirm that the user is an `admin`. It will then set the role for the selected user to the specified role.
+Start with the server action. The following `setRole()` action will first check the role of the current user using the `checkRoles()` helper to confirm that they are an `admin`. It will then set the specified role for the selected user.
 
 ```ts {{ filename: 'app/admin/dashboard/_actions.ts' }}
 'use server'
 
-import { checkRole } from '@/utils/roles'
+import { checkRole } from '../../../utils/roles'
```

Comment:
Next by default uses  the `@` for an alias import. I feel like that's a reasonable usage in our docs. If we aren't going to use that, then I think an absolute part (`app/utils/roles`) is probably more clear than a relative path.